src/pages/Contact/components/ContactDetailsCard/style.scss

```diff
@@ -0,0 +1,54 @@
+.details-card {
+    font-family: var(--font-secondary);
+    margin: 20px 0px;
+    padding: 5px 20px;
+    border: 1px solid var(--osweb-color-primary);
+    border-radius: 4px;
+    max-width: 500px;
+    min-height: 10rem;
+    max-height: 20rem;
+    width: calc(100% - 40px);
+    min-width: 250px;
+
+    @media (max-width: 1200px) {
+        margin: 10px 30px;
+        width:30%;
+      }
+  
+    &__content span {
+        color: var(--osweb-color-primary);
+        font-family: var(--font-secondary);
+        font-weight: 400;
+    }
+
+    &__content {
+        color: var(--osweb-color-custom-5);
+        font-family: var(--font-primary);
+        text-align: center;
+        font-size: 1.2rem;
+    }
```

Comment:
this can be replaced with

```
&__content {
    color: var(--osweb-color-custom-5);
    font-family: var(--font-primary);
    text-align: center;
    font-size: 1.2rem;

    & span {
      color: var(--osweb-color-primary);
      font-family: var(--font-secondary);
      font-weight: 400;
    }
  }
```